app/slashcommands-open/client/client.ts

```diff
@@ -0,0 +1,63 @@
+import { Meteor } from 'meteor/meteor';
+import { FlowRouter } from 'meteor/kadira:flow-router';
+
+import { roomTypes } from '../../utils/client';
+import { slashCommands } from '../../utils/lib/slashCommand';
+import { Subscriptions, ChatSubscription } from '../../models/client';
+
+function Open(_command: 'open', params: string, _item: Record<string, string>): void {
+	let room = params.trim();
+	const typeStr = room[0];
+	let type: string[] | null;
+	if (typeStr === '#') {
+		type = ['c', 'p'];
+	} else if (typeStr === '@') {
+		type = ['d'];
+	} else {
+		type = null;
+	}
```

Comment:
why did you removed the dict?

Reply:
Because it was giving type erros, so I would need to create an interface to the dict. I prefered to just create an if-else to the cases.

Reply:
It was giving that string can't be used to index the dict.